codebreaker_garage/lib/codebreaker_garage/console.rb

```diff
@@ -29,16 +30,20 @@ def want_to_save
       replay
     end
 
+    def correct_input
+      user_guessing = []
+      @user_tries_to_guess.each_char {|char| user_guessing.push char.to_i }
```

Comment:
In game.rb file, method guesser. I cannot use `zip` on a `:String`. I have converted it to array. For exsmple "1234" to [1,2,3,4]